test/addons/velero/start

```diff
@@ -17,8 +17,9 @@ def deploy(cluster):
         "velero",
         "install",
         "--provider=aws",
-        "--image=quay.io/nirsof/velero:v1.14.0",
-        "--plugins=quay.io/nirsof/velero-plugin-for-aws:v1.10.0",
+        "--image=quay.io/prd/velero:v1.16.1",
+        "--plugins=quay.io/prd/velero-plugin-for-aws:v1.12.0",
```

Comment:
We need to move these images to quay.io/ramendr, but using them from your repo is fine for now.

However the images need to be multi-arch images, since we use them also on macOS. The images in my repo are multi-arch.